packages/common/src/index.ts

```diff
@@ -675,11 +676,18 @@ export const getAxisName = ({
 export function getFieldMap(
     explore: Explore,
     additionalMetrics: AdditionalMetric[] = [],
+    tableCalculations: TableCalculation[] = [],
```

Comment:
Not sure how typescript allowed these changes since this method should only return field ( dimensions/metrics ).

We also have `getItemMap` that does what you implemented here.